src/App.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  background-color: whitesmoke;
+  box-sizing: border-box;
+  margin: 100px auto;
+  font-size: 20px;
+  @media only screen and (max-width: 600px) {
```

Comment:
Leave RWD for now. Do not remove it but you'll handle it when app will be ready.

Reply:
OK